korebuild-lock.txt

```diff
@@ -1,2 +1,2 @@
-version:2.1.0-preview1-15549
-commithash:f570e08585fec510dd60cd4bfe8795388b757a95
+version:2.1.0-preview1-15564
```

Comment:
@natemcmaster are these changes expected every now and then?

Reply:
sure, if you need a new version of build tools. The only reason to hold off for now is that the jump from 2.1.0-preview1-15549 to 2.1.0-preview1-15564 puts you ahead of aspnet/Universe

Reply:
cref https://github.com/aspnet/Universe/blob/dev/korebuild-lock.txt